source/ClientApp/src/app/events/event.component.ts

```diff
@@ -0,0 +1,39 @@
+import { Component } from '@angular/core';
+import { Event } from './Models/event.model';
+import { EventService } from './Services/event.service';
+
+@Component({
+  selector: 'app-event',
+  templateUrl: './event.component.html'
+})
+export class EventComponent {
+  public events: Event[];
+
+  constructor(private eventService: EventService) {
+    this.eventService.getEvents('jss94').subscribe(response => {
+      this.events = response;
+    });
+
+  }
+
+  createNewEvent(): void {
+    console.log('hello there');
+    const testEvent: Event = {
+      organizerId: 'jss94',
+      name: 'New Test Event',
+      description: 'My Test Event Description',
+      eventCreated: null,
+      eventDate: null,
+      eventId: null,
+      guestListId: null
```

Comment:
I believe I addressed this by adding in an eventId.  For the object to be created I need to specify an EventId, however, on create, it would be handled by the database.  Is there a better way to handle this in Angular as to not have to include it?